.github/workflows/dart.yml

```diff
@@ -54,9 +54,9 @@ jobs:
       - run: dart --version
       - uses: actions/checkout@v2
       - id: pkg_web_css_pub_get
-        name: "pkg/web_css; pub get --no-precompile"
+        name: "pkg/web_css; dart pub get --no-precompile"
```

Comment:
This is a generated file, as it there is some support for old versions this haven't been migrated yet, see `mono_repo`

Reply:
Ah Thank
@jonasfj